Extensibility/TemplatePack/ProjectTemplates/CSharp/Forms/PrismAutofacApp/AutofacApp/App.xaml.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Autofac;
+using Prism.Autofac;
+using Prism.Autofac.Forms;
+using $safeprojectname$.Views;
+
+namespace $safeprojectname$
+{
+    public partial class App : PrismApplication
+    {
+        public App(IPlatformInitializer initializer = null) : base(initializer) { }
+
+        protected override void OnInitialized()
+        {
+            InitializeComponent();
+
+            NavigationService.NavigateAsync("MainPage?title=Hello%20from%20Xamarin.Forms");
+        }
+
+        protected override void RegisterTypes()
+        {
+            var builder = new ContainerBuilder();
```

Comment:
In the case of the current Autofac, an error occurs unless you register ViewModel.
Is this something wrong with me?
If I am correct, I should fix AutofacExtensions.
It becomes a problem when registering View with ItemTemplate.